app/controllers/content_items_controller.rb

```diff
@@ -24,6 +25,11 @@ def filter
 
 private
 
+  def populate_filters
```

Comment:
I would not include the word `filters` in the controller. `filters` are view related

What about splitting up the method into:

```ruby
before_action :set_all_organisations, only: :index
before_action :set_all_taxonomies, only: :index